src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -523,10 +533,20 @@ private function addSerializerSection(ArrayNodeDefinition $rootNode)
     private function addPropertyAccessSection(ArrayNodeDefinition $rootNode)
     {
         $rootNode
+            ->beforeNormalization()
+                ->always(function ($v) {
+                    if (!isset($v['property_access'])) {
+                        $v['property_access']['enabled'] = true;
+                        @trigger_error('You must explicitly define wheter to enable "framework.property_access" or not. It will be disabled by default in 4.0', E_USER_DEPRECATED);
```

Comment:
I commented it for now as there are a lot of changes to make to fix all the deprecations in the core then ...